test/testvalueflow.cpp

```diff
@@ -7881,20 +7881,20 @@ class TestValueFlow : public TestFixture {
     }
 
     void valueFlowInc() {
-       const char *code;
-       std::list<ValueFlow::Value> values;
-        /*const*/ Settings s = settingsBuilder().library("std.cfg").build();
+        const char *code;
+        std::list<ValueFlow::Value> values;
+        /*const*/ Settings s = settingsBuilder().build();
```

Comment:
You can completely remove this variable. It is only necessary if you need to pass custom settings.